server/pkg/accounts/accounts.go

```diff
@@ -59,13 +62,9 @@ func CreateAccount(db database.Database, logger log.Logger, account *models.Acco
 	return nil
 }
 
-func validateAccount(account models.Account) error {
-	if account.Email == "" {
-		return fmt.Errorf(invalidAccountFormatter, "email")
-	}
-	if account.Username == "" {
-		return fmt.Errorf(invalidAccountFormatter, "username")
-	}
-
-	return nil
+func (account models.Account) validateAccount() error {
```

Comment:
This fails to build since isn't in account. I've been trying to migrate objects out of models -- can you move this one to the `accounts` package and update accordingly?

Reply:
what do you mean with this "This fails to build since isn't in account. " ?
Does Account struct is in models package?

Reply:
Correct. Because the account struct is in the models package and not the accounts package you can't add functions to it like you are doing here.

The account struct should be moved out of the models package and into the accounts package.

Reply:
Kindly check it again